internal/commands/push/inputs.go

```diff
@@ -47,17 +49,20 @@ func (i *inputs) Resolve(profile *user.Profile, ui terminal.UI) error {
 		searchPath = profile.WorkingDirectory
 	}
 
+	_, err := os.Stat(path.Join(profile.WorkingDirectory, i.LocalPath))
+	if os.IsNotExist(err) {
+		return errProjectInvalid{Path: i.LocalPath, Exists: false}
```

Comment:
[nit] can omit the `Exists: false` part here (that's just the zero value anyways)
```suggestion
		return errProjectInvalid{Path: searchPath, Exists: false}
```